docs/components/radios/test.html

```diff
@@ -11,6 +12,958 @@
+          :focus
+        </label>
+      </hx-radio-control>
+      <hx-radio-control>
+        <input type="radio" id="radSelected" checked />
+        <label for="radSelected">
+          <hx-radio></hx-radio>
+          :checked
+        </label>
+      </hx-radio-control>
+      <hx-radio-control>
+        <input type="radio" id="radRequiredC" required />
+        <label for="radRequiredC">
+          <hx-radio></hx-radio>
+          :required C (group size 1)
+        </label>
+      </hx-radio-control>
+      <hx-radio-control>
+        <input type="radio" id="radRequiredA" value="" name="testReq" required />
+        <label for="radRequiredA">
+          <hx-radio></hx-radio>
+          :required A
+        </label>
+      </hx-radio-control>
+      <hx-radio-control>
+        <input type="radio" id="radRequiredB" value="B" name="testReq" required />
+        <label for="radRequiredB">
+          <hx-radio></hx-radio>
+          :required B
+        </label>
+      </hx-radio-control>
+      <hx-radio-control>
+        <input type="radio" id="radInvalidNative" data-mock-invalid />
+        <label for="radInvalidNative">
+          <hx-radio></hx-radio>
+          :invalid
+        </label>
+      </hx-radio-control>
+      <hx-radio-control>
+        <input type="radio" id="radDisabled" disabled />
+        <label for="radDisabled">
+          <hx-radio></hx-radio>
+          [disabled]
+        </label>
+      </hx-radio-control>
+    </div>
+
+    <h2>Radio Button Groups</h2>
+    <h3>Radio Group with Labels</h3>
+    <div class="hxDiv">
+      <hx-radio-control>
+        <input type="radio" name="basic-radio-group" id="rad1" />
+        <label for="rad1">
+          <hx-radio></hx-radio>
+          Radio Control 1
+        </label>
+      </hx-radio-control>
+
+      <hx-radio-control>
+        <input type="radio" name="basic-radio-group" id="rad2" />
+        <label for="rad2">
+          <hx-radio></hx-radio>
+          Radio Button 2
+        </label>
+      </hx-radio-control>
+
+      <hx-radio-control>
+        <input type="radio" name ="basic-radio-group" id="rad3" />
+        <label for="rad3">
+            <hx-radio></hx-radio>
+            Radio Button 3
+          </label>
+      </hx-radio-control>
+    </div>
+  </section>
+
+  <hr />
+
+  <section>
+    <header>
+      <h1>Radio State Table</h1>
+      <h2>Optional - Enabled</h2>
+      <code>:enabled :unchecked</code>
+    </header>
+    <table>
+      <thead>
+        <tr>
+          <th></th>
+          <th>:valid</th>
+          <th>:invalid</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>(none)</td>
+          <td>
+            <hx-radio-control>
+              <input
+                type="radio"
+                id="radOptionalEnabledUncheckedNoneValid"
+              />
+              <label for="radOptionalEnabledUncheckedNoneValid">
+                <hx-radio></hx-radio>
+                :optional:enabled:unchecked:none:valid
```

Comment:
add focus styles in the tables on this page where we talked about on zoom!!

Reply:
done